packages/flutter/test/material/dropdown_test.dart

```diff
@@ -252,37 +284,50 @@ void main() {
   });
 
   testWidgets('Dropdown button aligns selected menu item', (WidgetTester tester) async {
-    final Key buttonKey = new UniqueKey();
-    final String value = 'two';
-
-    Widget build() => buildFrame(buttonKey: buttonKey, value: value);
-
-    await tester.pumpWidget(build());
-    final RenderBox buttonBox = tester.renderObject(find.byKey(buttonKey));
-    assert(buttonBox.attached);
-    final Offset buttonOriginBeforeTap = buttonBox.localToGlobal(Offset.zero);
-
-    await tester.tap(find.text('two'));
-    await tester.pump();
-    await tester.pump(const Duration(seconds: 1)); // finish the menu animation
+    for (TextDirection textDirection in TextDirection.values) {
```

Comment:
I've been using separate tests for the directions to make the failures more obvious.  When this test fails, the output won't tell you which of the directions failed (or if they both fail).  Consider extracting the body of the test into a function and having an RTL and LTR test call it.

Reply:
Good point. I inverted the for loop and the test declaration:

```dart
  for (TextDirection textDirection in TextDirection.values) {
    testWidgets('Dropdown button aligns selected menu item ($textDirection)', (WidgetTester tester) async {
```